platformcomponents/desktop/attachment-file-preview-thumbnail.json

```diff
@@ -0,0 +1,12 @@
+{
+  "attachment-file-preview-thumbnail": {
+    "sent": {
+      "comment": "Applied to file attachments",
+      "figma": "https://www.figma.com/file/NaNrfXjygZtRgMfHAFHjsp/Components---Windows?node-id=49674%3A114116",
+      "border": "@theme-outline-secondary-normal",
+      "overlay": "@theme-common-overlay-primary",
```

Comment:
Perfect, we just need to group things that are visually together
```
"overlay":{
     "background": "@theme-common-overlay-primary".
     "label": {
             "primary": "@theme-common-text-white",
             "secondary": "@theme-common-text-gray"
     }
}
```

Reply:
Pushed commit
[Updated overlay tree-structure in attachment-file-preview-thumbnail](https://github.com/momentum-design/tokens/pull/174/commits/3ba49befe7e26d359c44ab6b125389c92483feb6)
Changing this conversation to Resolved.